<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Level3</class>
 <widget class="QWidget" name="Level3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-21</x>
     <y>0</y>
     <width>1061</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="introPage">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="levelButton">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>410</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pictures/selectlevel.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>400</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;
font-size: 14;
background-color: rgb(186, 189, 182)</string>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>601</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(174, 167, 167, 57);
color:rgb(238, 238, 236);
border-radius: 10px;
font: 70 50pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Sandwich Stacking Game&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Memorizing Long Lists&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="reuben_intro">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>271</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bubble_1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>311</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
background-color: rgb(213, 248, 248);
color: rgb(85, 87, 83);
qproperty-alignment: AlignCenter;
border: 1px solid rgb(136, 138, 133)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Hi, I'm Reuben!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="factsPage">
    <widget class="QLabel" name="reubenFact">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>231</width>
       <height>341</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>821</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(211, 215, 207,0.5);
color: rgb(32, 74, 135);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Let's come up with a system!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Tip #1:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Create a linking system.&lt;br/&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Using visuals to link different objects in a list helps you connect each item as you recall them.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="header_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>821</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(211, 215, 207,0.7);
color:rgb(52, 185, 209);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#000000;&quot;&gt;How to Memorize Lists&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextBtn_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>440</y>
       <width>180</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pictures/next.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tutorialPage">
    <widget class="QLabel" name="header_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>821</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(211, 215, 207,0.7);
color:rgb(52, 185, 209);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#000000;&quot;&gt;HOW TO PLAY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextBtn_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>440</y>
       <width>180</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pictures/next.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="msg_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>821</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(211, 215, 207,0.5);
color: rgb(32, 74, 135);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Use the &lt;/span&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;left &lt;/span&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;and&lt;/span&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt; right arrow keys&lt;/span&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt; to catch falling food. Yum!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Tip #2:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Make it memorable&lt;/span&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Memorable mnemonics give you an easier time recalling any type of list. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Say the ingredients out loud with me!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="reubenTut">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>190</y>
       <width>241</width>
       <height>321</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <zorder>header_3</zorder>
    <zorder>msg_3</zorder>
    <zorder>reubenTut</zorder>
    <zorder>nextBtn_3</zorder>
   </widget>
   <widget class="QWidget" name="gamePage">
    <widget class="QPushButton" name="tutorialButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>270</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pictures/yellowbackground.png);
</string>
     </property>
     <property name="text">
      <string>Start Tutorial</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectlevel_button">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>470</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pictures/selectlevel.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="playNowButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pictures/yellowbackground.png);</string>
     </property>
     <property name="text">
      <string>Play Now</string>
     </property>
    </widget>
    <widget class="QLabel" name="sLabel">
     <property name="geometry">
      <rect>
       <x>922</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: solid 1px black;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;SCORE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="scoreLabel">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: solid 1px black;
background-color:rgba(174, 167, 167, 57);
font: 25 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="reubenLabel">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>304</y>
       <width>121</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="reubenTalk">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>190</y>
       <width>121</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(174, 167, 167, 57);
font: 25 12pt &quot;Ubuntu&quot;;color:white;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="livesLabel">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: solid 1px black;
background-color:rgba(174, 167, 167, 57);
font: 25 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="sLabel_2">
     <property name="geometry">
      <rect>
       <x>925</x>
       <y>65</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: solid 1px black;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;LIVES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="gameMessage">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>561</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 11pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;GAME OVER&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="tutMsg">
     <property name="geometry">
      <rect>
       <x>187</x>
       <y>360</y>
       <width>561</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black; font: 25 11pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="levelLabel">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>140</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(174, 167, 167, 57);
border: solid 1px black;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;LEVEL 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="helpMsg">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>561</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;help&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>graphicsView</zorder>
    <zorder>tutorialButton</zorder>
    <zorder>selectlevel_button</zorder>
    <zorder>playNowButton</zorder>
    <zorder>sLabel</zorder>
    <zorder>scoreLabel</zorder>
    <zorder>reubenLabel</zorder>
    <zorder>reubenTalk</zorder>
    <zorder>livesLabel</zorder>
    <zorder>sLabel_2</zorder>
    <zorder>gameMessage</zorder>
    <zorder>tutMsg</zorder>
    <zorder>levelLabel</zorder>
    <zorder>helpMsg</zorder>
   </widget>
   <widget class="QWidget" name="quizMsgPage">
    <widget class="QLabel" name="header">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>821</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(211, 215, 207,0.7);
color:rgb(52, 185, 209);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; color:#000000;&quot;&gt;EXERCISING YOUR MEMORY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="msg">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>821</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(211, 215, 207,0.5);
color: rgb(32, 74, 135);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Tip #3:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Give your brain a workout.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;One way you can improve you brain's ability to recall and build your memory is by exercising your brain muscles through challenges and stimulation. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextBtn">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>450</y>
       <width>180</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Pictures/next.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="reubenQuiz">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>191</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="foodImg">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>170</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="quizPage">
    <widget class="QSlider" name="timeSlider">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>420</y>
       <width>741</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="quiztimelabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>691</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(211, 215, 207,0.5);
color: rgb(32, 74, 135);font-size: 50; 
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Quiz Time!&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;What acronym did the sandwich ingredients create?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userInput">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>300</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Type here</string>
     </property>
    </widget>
    <widget class="QLabel" name="questionLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>691</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(211, 215, 207,0.5);
color: rgb(32, 74, 135);font-size: 50; 
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Time's Out&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
